trunk/source/BA_SiteRepository/Ultimate_E4.32_Java21.target

```diff
@@ -60,6 +58,24 @@
 		</location>
 		<location includeDependencyDepth="none" includeDependencyScopes="compile" includeSource="false" missingManifest="generate" type="Maven">
 			<dependencies>
+				<dependency>
+					<groupId>com.lmax</groupId>
+					<artifactId>disruptor</artifactId>
+					<version>3.4.4</version>
+					<type>jar</type>
+				</dependency>
+				<dependency>
+					<groupId>net.sf.trove4j</groupId>
+					<artifactId>trove4j</artifactId>
+					<version>3.0.3</version>
+					<type>jar</type>
+				</dependency>
```

Comment:
So I guess it turned out this dependency is still used after all?

Reply:
Yes, it is used, e.g. [here](https://github.com/ultimate-pa/ultimate/blob/0a57b278650c6e4f7c1dc59465c12955e1a35a6e/trunk/source/Library-Automata/src/de/uni_freiburg/informatik/ultimate/automata/nestedword/operations/optncsb/automata/AccGenBuchi.java#L33).